src/cbmc_starter_kit/template-for-repository/proofs/lib/summarize.py

```diff
@@ -116,4 +147,19 @@ def print_proof_results(out_file):
 
 if __name__ == '__main__':
     args = get_args()
-    print_proof_results(args.run_file)
+    with_aws = args.cloudfront_domain and args.s3_uri
+    without_aws = not args.cloudfront_domain and not args.s3_uri
+    if not with_aws and not without_aws:
```

Comment:
Nit (De Morgan's law): I think `if not (with_aws or without_aws)`, or TBH `if not (both_args or neither_arg)` reads more naturally

Reply:
Using first one